xbmc/network/WakeOnAccess.cpp

```diff
@@ -68,6 +73,117 @@ static unsigned long HostToIP(const std::string& host)
   return inet_addr(ip.c_str());
 }
 
+#define LOCALIZED(id) g_localizeStrings.Get(id)
+
+static void ShowDiscoveryMessage(const char* function, const char* server_name, bool new_entry)
+{
+  std::string message, heading = LOCALIZED(13033);
+
+  if (new_entry)
+  {
+    CLog::Log(LOGDEBUG, "%s - Create new entry for host '%s'", function, server_name);
+    message = StringUtils::Format(LOCALIZED(13035).c_str(), server_name);
+  }
+  else
+  {
+    CLog::Log(LOGDEBUG, "%s - Update existing entry for host '%s'", function, server_name);
+    message = StringUtils::Format(LOCALIZED(13034).c_str(), server_name);
+  }
+  CGUIDialogKaiToast::QueueNotification(CGUIDialogKaiToast::Info, heading, message, 4000, true, 3000);
```

Comment:
Interesting, I am not familiar with that, could you give me a pointer and I can investigate. Would it be reasonable to make a PR changing all use of 'kaitoast' in this module into that?